wiki/Help_centre/Upgrading_to_lazer/it.md

```diff
@@ -0,0 +1,290 @@
+---
+tags:
+  - game client
+---
+
+# Aggiornare a lazer
+
+*Avviso: Stiamo ancora cercando di bilanciare e di sistemare le meccaniche di gioco. Per ora, **I punteggi che sono stati effettuati su lazer non saranno permanenti**.*
+
+osu!(lazer) è il prossimo grande aggiornamento al client di osu!. È il frutto di diversi anni di lavoro dietro le quinte per reimplementare osu!.
+
+L'obiettivo finale é quello di rilasciare questa versione come aggiornamento che sostituirá completamente l'attuale versione esistente del gioco, ma c'è ancora del lavoro da fare prima che sia fattibile. Alla fine, saranno **i giocatori** a determinare quando ciò accadrà e continueremo a supportare la versione precedente fino a quando gli utenti non avranno effettuato il passaggio.
+
+"lazer" è un nome in codice e verrà eventualmente scartato una volta che diventerà la versione di rilascio principale del gioco. Il resto di questo documento si riferirà a osu!(lazer) come "lazer" e osu!(stable) come "stable" per semplicità.
+
+## Differenze tra stable e lazer
+
+Di seguito verrà rappresentata una lista dello **stato attuale** di lazer in confronto con stable. Bisogna tenere a mente che il progetto è in continua evoluzione — l'obiettivo finale è quello di implementare tutte le funzioni che i giocatori hanno richiesto nel corso degli anni.
+
+### Compatibilità e prestazioni
+
+| Funzionalità | stable | lazer |
+| :-- | :-- | :-- |
+| Windows 8.0 e versioni precedenti | ![Yes][true] | ![No][false] |
```

Comment:
the Yes and No and Partial should be translated

they appear if the image can't be loaded